tron/api/adapter.py

```diff
@@ -311,6 +313,12 @@ def get_runs(self):
     def get_max_runtime(self):
         return str(self._obj.max_runtime)
 
+    def get_expected_runtime(self):
+        return str(self._obj.expected_runtime)
+
+    def get_actions_expected_runtime(self):
+        return self._obj.action_graph.expected_runtime
```

Comment:
Does this need `str()`? Also, will this be a timedelta or the string that was passed in? If it's already a timedelta, might be easier to convert the timedelta to seconds here in the adapter, rather than doing it in the script.

Reply:
Yeah, it should have str(). I thought about and decided to keep it consistent with other timedelta objects, such as max_runtime.

Reply:
Ok, I change my mind. It's not consistent for timedelta objects either. So I will convert it to seconds in Tron rather than in the script.